cmd/sealos/cmd/delete.go

```diff
@@ -35,6 +35,10 @@ delete to nodes:
 delete to default cluster: 
```

Comment:
this tips can remove it, delete cmd can't delete cluster .add tips delete to masters